CHANGELOG.md

```diff
@@ -4,7 +4,55 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
-## [v2.0.0](https://github.com/voxpupuli/puppet-nscd/tree/v2.0.0) (2019-03-09)
```

Comment:
I wonder why this date was wrong previously??

Reply:
Summary 

Currently published with: `v2.0.0 (2019-03-09)`
CHANGELOG now has `v2.0.0 (2019-05-02)


```
2019-02-05  -  a72f8ac (modulesync) [blacksmith] Bump version to 2.0.1-rc0
2019-02-05  -  c3796ad (tag: v2.0.0) Merge pull request #49 from bastelfreak/rel200
2019-07-04  -  012baa8 Merge pull request #50 from voxpupuli/modulesync
2019-07-04  - 8eda5c1 modulesync 2.7.0
2019-09-03  - dfd122d release 2.0.0
```
So a delay between release MR being merged and the actual tagging.